### ISSUE TITLE
Specification of floating point number notation

### ISSUE THREAD

**Issue by nevrome** [poseidon-framework/poseidon-schema#44]
As discussed in https://github.com/poseidon-framework/published_data/pull/30 there are different ways of writing floating point numbers and we should consider specifying that for the POSEIDON.yml and .janno file.

**Comment by stschiff** [comment]
Right. I think [this here](https://en.wikipedia.org/wiki/IEEE_754#Basic_and_interchange_formats) could be an authoritative way to cite.

**Comment by nevrome** [comment]
At the moment the schema and all relevant software tools support a range of floating point number notations. I now think it would be unnecessarily restrictive to limit this to one.

It is annoying to encounter Git diffs with plenty of pointless changes after saving a .janno file with some spreadsheet software. But probably it would be even more painful for users to see package validation fail just because of a difference in notation.

I set this to **wontfix** for now.

**Comment by stschiff** [comment]
I agree!
